timescaledb/how-to-guides/hypertables/create.md

```diff
@@ -74,9 +85,79 @@ hypertable fails. For more information about setting up multi-node, see the
      SELECT create_distributed_hypertable('conditions', 'time', 'location');
      ```
 
+<highlight type="note"> 
+If your table has a `PRIMARY KEY` or `UNIQUE` constraint, that key or constraint
+must include all columns that are used to partition your hypertable. In this
+example, the key or constraint must include `time` and `location`.
+
+It may additionally include columns that aren't partitioning columns. For more
+information, see [the troubleshooting
+section](https://docs.timescale.com/timescaledb/latest/how-to-guides/hypertables/create/#primary-keys-and-unique-constraints).
+</highlight>
+
 </procedure>
 
+## Troubleshooting
+
+### Primary keys and unique constraints
+
+If your `PRIMARY KEY` or `UNIQUE` constraint doesn't contain all your
```

Comment:
I wonder if we should jsut have a brief section explaining that a PRIMARY KEY includes a UNIQUE constraint, so we can just refer to everything by that, but in case people don't know...

Reply:
Ohh good idea, the "key or constraint" thing is getting repetitive